config/source_input_no.go

```diff
@@ -0,0 +1,39 @@
+package config
```

Comment:
Just a thought, since there aren't really any functional differences between source_input_no.go and condition_no.go, how would you feel about creating a struct called `NoMonitorConfig` instead? Since both SourceInput and Condition are composed of the Monitor type, you could set the empty values of both to `NoMonitorConfig{}`

eg.
```GO
func EmptySourceInputConfig() SourceInputConfig {
	return &NoMonitorConfig{}
}

func isSourceInputEmpty(c SourceInputConfig) bool {
	_, ok := c.(*NoMonitorConfig)
	return ok
}
```

```GO
func EmptyConditionConfig() ConditionConfig {
	return &NoMonitorConfig{}
}
```

Reply:
Yea good point. I think the NoMonitorConfig is a great idea - reduces code to maintain